### DISCUSSION TITLE
Theme for NeoVim

### DISCUSSION THREAD

**Discussion by marilira** [daltonmenezes/aura-theme#125] [Port request]
I use the theme on vscode and would also to use on my Neovim :)

**Comment by daltonmenezes** [comment]
Hi @marilira , can you provide me some infos?

1. Neovim uses the same vim colorschemes?
2. There Is no difference about neovim and vim themes?
3. What steps I need to apply a theme on neovim?

**Comment by marilira** [comment]
I haven't been using Neovim / vim long enough, but what do I know:

1. At the moment I am using [dracula for vim](https://github.com/dracula/vim) and works properly on Neovim.
2. I see other colorschemes for Neovim  written in Lua [like this one](https://github.com/Mofiqul/dracula.nvim)
3. I use vim-plug to manage the plugins, so in my config file (on Windows it's: `~\AppData\Local\nvim\init.vim`) it looks like this:

```vim
call plug#begin('~\AppData\Local\nvim-data\plugged')

    Plug 'dracula/vim', {'as': 'dracula'}
    
call plug#end()

colo dracula "set the theme
```

I hope I have helped in something.

**Comment by ashishbinu** [comment]
Hey, I can give it a shot. I will try doing both vim and neovim themes. As, most of the stuff would be same.

**Comment by daltonmenezes** [comment]
Hi @ashishbinu , it would be great for sure. Thanks, just remember, one PR per port, let me know if you need some help and use vscode port as the base to put the colors in a right way. 💜

**Comment by ashishbinu** [comment]
> Hi @ashishbinu , it would be great for sure. Thanks, just remember, one PR per port, let me know if you need some help and use vscode port as the base to put the colors in a right way. :purple_heart:

Yeah sure, I will keep in mind.

**Comment by daltonmenezes** [comment]
@ashishbinu thanks, just one more thing, don't forget to read the [documents](https://github.com/daltonmenezes/aura-theme#documentation) and comment the #56 issue requesting to work on this port, so I can assign you to that issue. 💜

**Comment by kavinvalli** [comment]
Hi, can someone please tell the status of this port @ashishbinu

**Comment by ashishbinu** [comment]
Sorry, @daltonmenezes I am currently busy with some issues in the real life. I was not able to devote time for this. I would be more busy coming months too. I am unassigning myself from this issue, hope someone else could finish this port.

**Comment by JLighter** [comment]
I created a repository related to this port.
I started from gruvbox neovim's theme which uses lush as theme framework.
Feel free to contribute by opening a PR, it'll be awesome !

**Comment by baliestri** [comment]
I started creating a port for this theme. I didn't use any specific framework, everything was based on tokyonight and neon themes. It's still not 100% compatible with Aura colors, I accept help or issues pointing out where I should change the colors.

Installation method only available for packer.nvim, please open discussions about other plugin managers.

https://github.com/baliestri/aura-theme/tree/feat/neovim-port/packages/neovim

**Reply by kavinvalli** [reply]
How complete is it @baliestri?

**Reply by baliestri** [reply]
> How complete is it @baliestri?

Just the basics, I've been doing other things, so it's stopped on my machine, correcting the colors in some places, soon I'll do another push
![image](https://user-images.githubusercontent.com/46748996/159706445-f7f47c7f-5c86-49ba-bc6b-e01513092c94.png)

**Comment by helbing** [comment]
@daltonmenezes, I'm working on this now, can you give all the accent colors, not through function, there is no color library in Lua.

<img width="636" alt="image" src="https://user-images.githubusercontent.com/7275620/194470937-39a0c129-002b-410f-a09a-c745ce7b8511.png">

**Comment by helbing** [comment]
Hi guys, It was done. https://github.com/helbing/aura.nvim
